static/js/components/CommentTree.js

```diff
@@ -130,9 +125,9 @@ export default class CommentTree extends React.Component<*, *> {
               >
                 <a href="#">reply</a>
               </div>}
-            {isModerator && report !== undefined
+            {comment.num_reports
```

Comment:
we may want to add a quick test just checking that `Reports: #` doesn't show up when `comment.num_reports` is null